sda-download/api/sda/sda_test.go

```diff
@@ -354,21 +356,13 @@ func TestDownload_Fail_UnencryptedDownloadNotAllowed(t *testing.T) {
 	expectedStatusCode := 400
 	expectedBody := []byte("downloading unencrypted data is not supported")
 
-	if response.StatusCode != expectedStatusCode {
-		t.Errorf("TestDownload_Fail_UnencryptedDownloadNotAllowed failed, got %d expected %d", response.StatusCode, expectedStatusCode)
-	}
-	if !bytes.Equal(body, []byte(expectedBody)) {
-		// visual byte comparison in terminal (easier to find string differences)
-		t.Error(body)
-		t.Error([]byte(expectedBody))
-		t.Errorf("TestDownload_Fail_UnencryptedDownloadNotAllowed failed, got %s expected %s", string(body), string(expectedBody))
-	}
+	assert.Equal(t, expectedStatusCode, response.StatusCode, "Unexpected status code from download")
+	assert.Equal(t, expectedBody, body, "Unexpected body from download")
 
 	// Test downloading of unencrypted file from the s3 endpoint, should fail
-	// encrypted
 	w = httptest.NewRecorder()
 	c, _ = gin.CreateTestContext(w)
-	c.Request = &http.Request{Method: "GET", URL: &url.URL{Path: "/s3/somepath", RawQuery: "filename=somepath"}}
+	c.Request = &http.Request{Method: "GET", URL: &url.URL{Path: "/mocks3/somepath", RawQuery: "filename=somepath"}}
```

Comment:
Is this supposed to be `mocks3` ?

Reply:
Stupid tests ignore inputs..

Reply:
I just used the same naming as elsewhere in the tests with similar code. The tests could use a refactoring at some point but not here :smile: